src/ui/Button/Button.tsx

```diff
@@ -74,6 +74,16 @@ export const Button = styled('button', {
         lineHeight: '$shorter',
         borderRadius: '$3',
       },
+      smaller: {
```

Comment:
this should probably be called something like `icon` or `iconOnly`, since that's what you're using it for. SInce you're setting the width as well as the height, it's not just smaller.